website/content/commands/server-upgrade.mdx

```diff
@@ -27,13 +27,17 @@ Usage: `waypoint server upgrade [options]`
 
 - `-auto-approve` - Confirm server upgrade.
 - `-context-name=<string>` - Waypoint server context to upgrade.
-- `-platform=<string>` - Platform to upgrade the Waypoint server from.
+- `-platform=<string>` - Platform to upgrade the Waypoint server from, defaults to the platform stored in the context
```

Comment:
nit: the original should say `from. Defaults to the platform stored in the context.